examples/authentication/restapi/embedded_spec.go

```diff
@@ -46,33 +46,33 @@ func init() {
             "name": "info",
             "in": "body",
             "schema": {
-              "$ref": "#/definitions/social_id"
```

Comment:
Example of issue reported in #1816 
Just by regenerating the examples from master

Reply:
Thanks for reporting this